migrations/change_Page_translation.js

```diff
@@ -0,0 +1,8 @@
+import translateDeepL from '../src/utils/translate-deepl.js';
+
+export default async function (block) {
+  await translateDeepL({
+    block,
+    fields: ['title', 'additionalSameAsLinks'],
```

Comment:
@Obilarius Does it make sense to translate the `additionalSameAsLink`?
Personally, I don't think so and would propose to remove that field.